app/app.go

```diff
@@ -180,24 +178,30 @@ func NewXplaApp(
 	legacyAmino.RegisterConcrete(&ethermintsecp256k1.PubKey{}, ethermintsecp256k1.PubKeyName, nil)
 	legacyAmino.RegisterConcrete(&ethermintsecp256k1.PrivKey{}, ethermintsecp256k1.PrivKeyName, nil)
```

Comment:
ethermint's(legacy) feemarket and evm types also need to be registered. 
This causes problems when querying existing proposals and txs.

Reply:
[5afa926](https://github.com/xpladev/xpla/pull/174/commits/5afa926b466e1d374afabad46766689ab53c2024)